src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell.component.tsx

```diff
@@ -1,16 +1,25 @@
-import React from "react";
-import { ColorHelper } from "../../../../../../helpers/colors/color.helper";
+import React, { useEffect } from "react";
 import { CellColorSet } from "../../../../../../helpers/colors/cell-color-set.model";
 import { BaseCellInputComponent, BaseCellInputOptions } from "./base-cell-input.component";
 import { VerboseDate, WeekDay } from "../../../../../../common-models/month-info.model";
 import { TranslationHelper } from "../../../../../../helpers/tranlsations.helper";
+import { useDrag, useDrop } from "react-dnd";
+import classNames from "classnames/bind";
+import { getEmptyImage } from "react-dnd-html5-backend";
 
 export enum CellManagementKeys {
   Enter = "Enter",
   Escape = "Escape",
 }
 
+const PivotCellType = "Cell";
+export interface PivotCell {
+  type: string;
+  rowIndex: number;
+  cellIndex: number;
+}
```

Comment:
Ten interfejs jest potrzebny do komunikacji za pomocą biblioteki której użyłem do implementacji przeciągania.
Dane które opisuje - to cell od którego zaczęto przeciąganie.
Przekazuję go z tego poziomu, ponieważ gdybym pamiętać pivot, to musiałbym zmieniać stan sekcji, przez wszystkie jej dzieci renderowałyby się od nowa